challenges/functions.js

```diff
@@ -7,18 +7,31 @@
   * In the body of the function return the callback with the two parameters that you created
 */
 
+const consume = function(arg1, arg2, callback){
+  return callback(arg1, arg2);
+}
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
 
+const add = function(arg1, arg2){
+  return arg1 + arg2;
+}
+const multiply = function(arg1, arg2) {
+  return arg1 * arg2;
+}
+
+const greeting = function(firstName, lastName) {
+  return `Hello ${firstName} ${lastName}, nice to meet you!`;
+}
 
 /* Step 3: Check your work by uncommenting the following calls to consume(): */
-//consume(2,2,add); // 4
-// consume(10,16,multiply); // 160
-// consume("Mary","Poppins", greeting); // Hello Mary Poppins, nice to meet you!
+console.log(consume(2,2,add)); // 4
+console.log(consume(10,16,multiply)); // 160
+console.log(consume("Mary","Poppins", greeting)); // Hello Mary Poppins, nice to meet you!
 
```

Comment:
This is a good habit to pick up from the instructor, console.log() everything to help you find bugs.